lib/galaxy/model/migrations/util.py

```diff
@@ -1,6 +1,38 @@
+import logging
+from contextlib import contextmanager
+
 from alembic import op
+from sqlalchemy.exc import (
+    OperationalError,
+    ProgrammingError,
+)
+
+log = logging.getLogger(__name__)
 
 
 def drop_column(table, column):
     with op.batch_alter_table(table) as batch_op:
         batch_op.drop_column(column)
+
+
+@contextmanager
+def ignore_add_column_error(table, column):
```

Comment:
We've always been ignoring this unconditionally in the past, this looks like a good idea.

Reply:
Case by case as you've done here is the right thing, don't think that warrants an extra option.